Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -82,6 +88,20 @@ public DriveSubsystem(WPIFactory factory, XPropertyManager propManager, ShiftSub
         previousRightTicks = rightDrive.getPosition();
         leftInchesTraveled = 0;
         rightInchesTraveled = 0;
+        
+        drivingStateLatch.addObserver((Observable o, Object arg) -> {
+            if(arg instanceof EdgeType) {
+                EdgeType edge = (EdgeType)arg;
+                String type = "";
+                if(edge == EdgeType.RisingEdge) {
+                    type = "starting";
+                }
+                else if(edge == EdgeType.FallingEdge) {
+                    type = "stopping";
+                }
+                log.info(String.format("Drive %s left: %.2f, right: %.2f", type, this.leftDrive.get(), this.rightDrive.get()));
```

Comment:
What's the use of the raw values? This only runs when the state changes from below 0.01 to above that threshold, so these numbers will generally be one 0.01 and the other the arbitrary position that it was when the other started moving.